### ISSUE TITLE
Numeric values from TextField

### ISSUE THREAD

**Issue by rhard** [TheComputerM/svelte-materialify#105]
Is there any way how to receive the number from TextField instead of string?
The next code returns string:

```js
let value = 0;
...
<TextField type="number" bind:value={value}>Numeric field</TextField>
```

**Comment by Florian-Schoenherr** [comment]
We would need to change from binding to using `on:input` to do that.
Would you mind doing this instead?
```svelte
<TextField on:input={(e) => value = +e.target.value}>Numeric field</TextField>
```

**Comment by rhard** [comment]
Thanks for the suggestion. I currently fixed it by using:

```js
    $: nbrSlowInterval = Number(strSlowInterval);
    $: nbrWeatherInterval = Number(strWeatherInterval);
```

**Comment by Florian-Schoenherr** [comment]
@rhard keep it DRY 😉 
```svelte
const saveAsNumber= (e) => value = Number(e.target.value);
<TextField on:input={saveAsNumber}>SlowInterval</TextField>
<TextField on:input={saveAsNumber}>WeatherInterval</TextField>
```
But you probably want native input-number functionality (like these little arrows), right?

**Comment by rhard** [comment]
Exactly! I was expecting this is already there like in others frameworks by default.

**Comment by Florian-Schoenherr** [comment]
TextField uses `<input type="text" bind:value>` inside, sadly `<input {type} bind:value>` is not possible, svelte forbids it.
We could change to use `<input {type} {value} on:input={(e) => parse(type, e.target.value)}>` or something like that. Might be less performant.. probably gonna add `<NumberField/>` instead, because we will also have `<DatePicker/>`.

For now, maybe it would look more "material" to use a Slider instead.

**Comment by frankieali** [comment]
Can't you use `e.target.valueAsNumber` when `type="number"` is set?
